src/commands.ts

```diff
@@ -8,25 +8,30 @@ import * as pify from 'pify';
 import * as vscode from 'vscode';
 import Config from './config';
 import Utils from './utils';
+import { IContextMenuPaths } from './types';
+
 
 /* COMMANDS */
 
-async function open ( basePath ) {
+async function open ( passedBasePath: IContextMenuPaths | undefined ) {
+  let basePath: string = '';
 
-  basePath = basePath && !_.isString ( basePath ) ? basePath.fsPath : basePath;
+  if (passedBasePath && !_.isString( passedBasePath )) {
+    basePath = passedBasePath.fsPath;
+  }
 
   if ( basePath && !await Utils.folder.is ( basePath ) ) return vscode.window.showErrorMessage ( 'The target must be a directory, not a file' );
 
   const config = Config.get ();
 
-  let includeGlob = await vscode.window.showInputBox ({
+  let includeGlob: string = await vscode.window.showInputBox ({
```

Comment:
I think `vscode.window.showInputBox` returns `Promise<string | undefinded>`, not `Promise<string>`